template/.devcontainer/manual-setup-deps.py

```
@@ -0,0 +1,120 @@
+        else:
+            raise NotImplementedError(f"Package manager {self.package_manager} is not supported")
+
+
+def main():
+    args = parser.parse_args(sys.argv[1:])
+    is_windows = platform.system() == "Windows"
+    uv_env = dict(os.environ)
+    uv_env.update({"UV_PYTHON_PREFERENCE": "only-system", "UV_PYTHON": args.python_version})
+    skip_check_lock = args.skip_check_lock or args.optionally_check_lock
+    if args.skip_check_lock and args.optionally_check_lock:
+        print("Cannot skip and optionally check the lock file at the same time.")
```

Ruff (0.12.2)

`print` found

(T201)